nestkernel/syn_id_delay.h

```diff
@@ -105,6 +105,11 @@ struct SynIdDelay
     return disabled;
   }
 };
+
+//!< check legal size
```

Comment:
Doxygen documentation placed before the thing they document should start with `//!` only.